docs/product/issues/ownership-rules/index.mdx

```diff
@@ -27,19 +27,12 @@ Types of matches available:
 
 The general format of a rule is: `type:pattern owners`
 
-`type`
-
-Can be either `path`, module, `url`, or `tags.TAG_NAME`, which matches the specified tag.
-
-`pattern`
-
-The pattern you're matching on. For example, `src/javascript/*` for `path`, `[https://www.example.io/checkout](https://www.example.io/checkout)` for `url`, or `Chrome 81.0.*` for `tags.browser`.
-
-`pattern` matching supports unix-style [glob syntax](<https://en.wikipedia.org/wiki/Glob_(programming)>). For example, add `*` to match anything and `?` to match a single character. _This is not a regex._
-
-`owners`
-
-The owner, or list of owners, identified by either the email of the Sentry user or the name of the team, prefixed with `#`. Thus, the owner may be example@company.com or example1@company.com example2@company.com or `#backend-team`. A space separates each owner in the list of owners.
+ - **`type`** - Can be either `path`, `module`, `url`, or `tags.TAG_NAME`, which matches the specified tag.
+ - **`pattern`** - The pattern you're matching on, which supports unix-style [glob syntax](<https://en.wikipedia.org/wiki/Glob_(programming)>). For example, add `*` to match anything and `?` to match a single character. _This is not a regex._ For example:
+   - `src/javascript/*` for a `path` type
+   - `https://www.example.io/checkout` for a `url` type
+   - `Chrome 81.0.*` for a `tags.browser` type
```

Comment:
Could you add an additional example showing how to use `?` and maybe for a `module` type?